README.md

```diff
@@ -12,60 +12,58 @@ platform.
 
```

Comment:
By running *uniqush* on server side -> "By running `uniqush-push` on the server side?